DependencyInjection/DoctrineExtension.php

```diff
@@ -230,6 +230,10 @@ protected function loadDbalConnection($name, array $connection, ContainerBuilder
                 $connection['mapping_types'],
             ))
         ;
+
+        if (!empty($connection['use_savepoints'])) {
+            $def->addMethodCall('setNestTransactionsWithSavepoints', array(true));
```

Comment:
Please use the value of ``$connection['use_savepoints']`` to set the argument, to make the code less fragile.